home01/js/script_7.js

```diff
@@ -0,0 +1,4 @@
+// 7.Пользователь указывает объем флешки в Гб. Программа должна посчитать сколько файлов размером в 820 Мб помещается на флешку.
+const sizecont = 1024;
```

Comment:
такие константы часто виделяют с помощью UPPER_CASE
```suggestion
const MEGABYTES_IN_GIGABYTE = 1024;
```